hack/test-integration-setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -euo pipefail
+
+KUBEUSER=${KUBEUSER:-kubeadmin}
+
+oc patch configs.imageregistry.operator.openshift.io/cluster --patch '{"spec":{"defaultRoute":true}}' --type=merge
+REGISTRY=$(oc get route default-route -n openshift-image-registry -o json | jq -r .spec.host)
```

Comment:
I think you can use here the jsonpath instead of `-o json | jq -r ...`

Reply:
done